include/dfu/fmfu_fdev.rst

```diff
@@ -0,0 +1,47 @@
+.. _lib_fmfu_fdev:
+
+Full modem firmware update from flash device
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The fmfu_fdev library provides an API for applying a full modem firmware update from a flash device.
+
+Functionality
+*************
+
+The modem firmware and metadata stored in the flash device must conform to a specific serialization format.
+For more information, see :ref:`lib_dfu_target_full_modem_update`.
+
+The serialized modem firmware contains the hash of the firmware and a signature.
+These fields are used to pre-validate the modem firmware before it is programmed to the modem, ensuring that the data about to be written corresponds to the data that have been signed.
+Once the modem firmware is pre-validated, it is written to the modem using the :file:`nrf_fmfu.h` API.
+
+Serialization
+*************
+
+The modem firmware is serialized using the following CDDL scheme.
+
+.. literalinclude:: ../../subsys/dfu/fmfu_fdev/cddl/modem_update.cddl
+    :language: c
+
+
+A generated decoder (:file:`modem_update_decode.c`) is used for parsing data serialized with this format.
+
+Prerequisites
+*************
+
+* The serialized modem firmware must be stored in the flash device.
+* The modem library must be de-initialized.
```

Comment:
and put into DFU_MODE

Reply:
I say that it must be put in DFU mode instead, that requires that it is de-initialized first.